SmartDeviceLinkTests/RPCSpecs/RequestSpecs/SDLSetGlobalPropertiesSpec.m

```diff
@@ -89,6 +93,28 @@
         expect(testRequest.keyboardProperties).to(beNil());
         expect(testRequest.userLocation).to(beNil());
     });
+
+    it(@"Should return new init", ^ {
```

Comment:
It looks like there were never tests written for the other inits. Can you add a test for each `init` that is missing a test. The **SDLAppServiceManifestSpec** is a good example of how we add tests for each `init` and how to suppress warnings for deprecated inits.